front/app/api/projects/__mocks__/useProjects.ts

```diff
@@ -72,11 +72,11 @@ export const project1: IProjectData = {
         },
         down: {
           enabled: false,
-          disabled_reason: 'downvoting_disabled',
+          disabled_reason: 'disliking_disabled',
         },
       },
       // MISMATCH: this attribute doesn't exist on our type
-      // comment_voting_idea: {
+      // comment_reacting_idea: {
```

Comment:
Yes it does exist in the front-end - will look at uncommenting it

Reply:
Actually this looks like it will be quite a bit of work to get it working in the way it is intended, so not worth doing in this PR and potentially causing issues